templates/zerver/help/mute-a-user.md

```diff
@@ -68,6 +69,16 @@ have the following effects:
 
 {end_tabs}
 
+### Re-hide revealed messages of muted users
```

Comment:
How about, "Re-hide a message that has been revealed"?

Reply:
Posted a [comment](https://chat.zulip.org/#narrow/stream/101-design/topic/Hiding.20user-muted.20messages/near/1184731) about it on chat.zulip.org.

Reply:
Made these changes.